roles/tang/vars/Debian.yml

```diff
@@ -0,0 +1,5 @@
+---
+apt_cache_valid_time: 3600
```

Comment:
Where is this variable used? Seems it isn't, then maybe these distro-specific variables aren't necessary anymore?